doc/user-guide.md

```diff
@@ -61,7 +61,7 @@ By default this will be the namespace that the operator is running in. To watch
 mgr, err := manager.New(cfg, manager.Options{Namespace: ""})
 ```
 
-**// TODO:** Doc on manager options(Sync period, leader election, registering 3rd party types)
+**// TODO:** Doc on leader election
 
 ## Add a new Custom Resource Definition
 
```

Comment:
There is a typo below this line, it should be `Add a new Custom Resource Definition(CRD)`